### PR TITLE
Posts and Deletes for projects and completed climbs

### PR DESCRIPTION
**Pull request by rachelbock**
- Updated the local ip address in home page
- Updated ClimbsRecyclerViewAdapter to have contain an onclick listener for the project checkbox and the completed climbs
  checkbox. Also added listener interfaces to return the climb id and boolean value to be used in the climbs Fragment and
  Projects Fragment to update the database.
- Updated ArrayList to List in the WallsRecyclerViewAdapter and WallSectionfragment.
- Added a POST and DELETE for projects and a POST and DELETE for completed climbs to the ClamberService interface
- Added the UserClimbDataRequest class to hold the username and climb id value. Used to pass that information to the
  database.
- modified the climbs and projects fragment to include POST and DELETE for projects and completed climbs.

### CODE REVIEW

File: app/src/main/java/com/rage/clamber/Fragments/ClimbsFragment.java [hunk 1]
```diff
@@ -130,4 +133,112 @@ public void onFailure(Call<List<Climb>> call, Throwable t) {
...
+            final Call<Project> createProjectCall = clamber.createProject(request);
+            createProjectCall.enqueue(new Callback<Project>() {
+                @Override
+                public void onResponse(Call<Project> call, Response<Project> response) {
+                    Log.d(TAG, "Successfully set up project for user: " + mainUser.getUserName());
+                }
+
+                @Override
+                public void onFailure(Call<Project> call, Throwable t) {
+                    Log.d(TAG, "Failed to set up project for user:" + mainUser.getUserName());
```
Comment (nonvirtualthunk) [inline app/src/main/java/com/rage/clamber/Fragments/ClimbsFragment.java]: include `t` to the `Log.d` call to ensure you have the stack trace if you ever hit this situation

File: app/src/main/java/com/rage/clamber/Fragments/ClimbsFragment.java [hunk 2]
```diff
@@ -130,4 +133,112 @@ public void onFailure(Call<List<Climb>> call, Throwable t) {
...
+                        Log.d(TAG, "Successfully removed project for user: " + mainUser.getUserName());
+                    } else {
+                        Log.d(TAG, "Failed to delete project for user: " + mainUser.getUserName());
+
+                    }
+                }
+
+                @Override
+                public void onFailure(Call<Boolean> call, Throwable t) {
+                    Log.d(TAG, "Failed to delete project for user: " + mainUser.getUserName());
```
Comment (nonvirtualthunk) [inline app/src/main/java/com/rage/clamber/Fragments/ClimbsFragment.java]: here as well

**Comment by nonvirtualthunk** [comment]
Good looking code. There's some code duplication but, as we discussed, resolving that will be part of the next PR.

File: app/src/main/java/com/rage/clamber/Fragments/ClimbsFragment.java (hunk 1 above)
Comment (rachelbock) [inline app/src/main/java/com/rage/clamber/Fragments/ClimbsFragment.java]: ok

File: app/src/main/java/com/rage/clamber/Fragments/ClimbsFragment.java (hunk 2 above)
Comment (rachelbock) [inline app/src/main/java/com/rage/clamber/Fragments/ClimbsFragment.java]: k

**Pull request merged**